lib/peep_repository.rb

```diff
@@ -0,0 +1,37 @@
+require_relative './peep'
+
+class PeepRepository
+  def all 
+    sql = "SELECT * FROM peeps;"
+    result = DatabaseConnection.exec_params(sql, [])
+    result.map do |record|
+      peep = Peep.new
+      peep.id = record['id'].to_i
+      peep.content = record['content']
+      peep.post_time = record['post_time']
+      peep.user_id = record['user_id'].to_i
+      peep
+    end
+  end
+
+  def create(peep)
+    sql = 'INSERT INTO peeps (content, post_time, user_id) VALUES ($1, $2, $3) RETURNING id;'
+    param = [peep.content, peep.post_time, peep.user_id]
+    results = DatabaseConnection.exec_params(sql, param)
+    peep.id = results.first['id'].to_i
+    return peep
```

Comment:
I think `return peep` isn't needed here - this could be causing the issue with not being able to create a peep in app.rb. 

I think you can just do `return nil` as this method's job is just to insert a peep into the database.